tests/Unit/SearchEngine/Driver/Elasticsearch/ElasticSearchIndexerTest.php

```diff
@@ -142,18 +146,16 @@ public function testBuildIndex(): void
                         '_id' => 'some_id'
                     ]],
                     ['type' => [
-                            TaoOntology::CLASS_URI_ITEM
+                        TaoOntology::CLASS_URI_ITEM
                     ]],
                 ]
             ])
             ->willReturn(['bulk_response']);
 
-        $this->logger->expects($this->at(2))
-            ->method('debug')
-            ->with('Processed 1 items (no exceptions, no skipped items)');
-
+        // Call the method under test
         $count = $this->sut->buildIndex($iterator);
 
+        // Verify the count
```

Comment:
What count? Why is this comment necessary
 ```suggestion
```

Reply:
Why not?

Reply:
Because this method is self explanatory and your comment is only polluting code